### PR TITLE
div. Features

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/controllers/de/laser/SurveyController.groovy [hunk 1]
```diff
@@ -889,7 +894,7 @@ class SurveyController {
         result.selectedParticipants = surveyService.getfilteredSurveyOrgs(surveyOrgs.orgsWithoutSubIDs, fsq.query, fsq.queryParams, params)
         result.selectedSubParticipants = surveyService.getfilteredSurveyOrgs(surveyOrgs.orgsWithSubIDs, fsq.query, fsq.queryParams, params)
 
-        result.selectedCostItemElement = params.selectedCostItemElement ?: RefdataValue.getByValueAndCategory('price: consortial price', RDConstants.COST_ITEM_ELEMENT).id.toString()
+        result.selectedCostItemElement = params.selectedCostItemElement ? params.selectedCostItemElement.toString() : RefdataValue.getByValueAndCategory('price: consortial price', RDConstants.COST_ITEM_ELEMENT).id.toString()
```
Comment (croqueGrec09) [inline grails-app/controllers/de/laser/SurveyController.groovy:897]: id or value?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/controllers/de/laser/SurveyController.groovy (hunk 1 above)
Comment (Moetez16) [reply grails-app/controllers/de/laser/SurveyController.groovy:897]: id

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**